gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/idl/org.apache.gobblin.service.flowexecutions.restspec.json

```diff
@@ -22,7 +22,6 @@
       "name" : "latestFlowExecution",
       "parameters" : [ {
         "name" : "flowId",
-        "doc" : "Retrieve the most recent matching FlowExecution(s) of the identified FlowId",
```

Comment:
Is removing the doc in these intentional?

Reply:
This file is auto-generated, and I guess this line was added to it manually previously. I've added javadocs to java interface, and those doc comments came back.